app/controllers/users_controller.rb

```diff
@@ -2,6 +2,8 @@ class UsersController < ApplicationController
   include PaginationUtility
   include ImagesUtility
 
+  LOG_FILE = 'log/users.log'
```

Comment:
Should preface the log file name with the Rails environment (ex: 'test', 'development', etc.)
This will help tremendously during testing: we will not have to read through 1 file to figure out which entries were made during a test (rspec or feature).
It will be clear exactly which entries were made during testing.  

As an example, here's the code I used for constructing the log file name in PR 445 (Observer/Requirements spike) for the MembershipUpdater:

```
 File.join(LOG_DIR, "#{Rails.env}_#{self.class}.log")
```
You can obviously replace {self.class} with the name you're using ('users').

Reply:
Great point - fixed.